deps/libgit2.mk

```diff
@@ -84,6 +84,11 @@ $(LIBGIT2_SRC_PATH)/libgit2-mbedtls-writer-fix.patch-applied: $(LIBGIT2_SRC_PATH
 		patch -p1 -f < $(SRCDIR)/patches/libgit2-mbedtls-writer-fix.patch
 	echo 1 > $@
 
+$(LIBGIT2_SRC_PATH)/libgit2-mbedtls-verify.patch-applied: $(LIBGIT2_SRC_PATH)/source-extracted
```

Comment:
How do I accomplish that? Do I just add the following?
`.../source-extracted | $(LIBGIT2_SRC_PATH)/libgit2-mbedtls.patch-applied`